example/OptionalExample.elm

```diff
@@ -67,12 +67,12 @@ streetNameOfAddress =
 
 regionOfPlace : Optional Place String
 regionOfPlace =
-    addressOfPlace `Monocle.Optional.compose` regionOfAddress
+    Monocle.Optional.compose addressOfPlace regionOfAddress
```

Comment:
Should the arguments here be flipped? To the effect of:

```
addressOfPlace
|> Monocle.Optional.compose regionOfAddress
```

Or, for obvious naming:

```
addressOfPlace
|> Monocle.Optional.andThen regionOfAddress
```